src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,44 @@
+import Backdrop from "../Backdrop/Backdrop";
+
+interface ModalProps {
+  children?: React.ReactNode;
+  show: boolean;
+  title: string;
+  maxWidth?: number;
+  onCloseRequest: () => void;
+}
+function Modal({
+  children,
+  show,
+  title = "Confirm Deletion",
+  onCloseRequest,
+}: ModalProps) {
+  const handleKeydown = (event: React.KeyboardEvent) => {
```

Comment:
en lugar del handlerevent
```suggestion
   useEscapeKey(onCloseRequest);
```